### DISCUSSION TITLE
Rename Windows exe to godot.exe and godot_console.exe for Scoop

### DISCUSSION THREAD

**Discussion by HotariTobu** [godotengine/godot-proposals#10718] [Platforms]
I propose changing Godot executable filenames in ZIP for Windows so that versions or architectures are not included.

## Why?

Godot installed via Scoop cannot be used as a command line tool:

![Animation](https://github.com/user-attachments/assets/d6fc3469-56e2-4637-887b-cff0ce54791f)

In the current Scoop manifest of Godot, pre-install commands remove the console version exe file and rename the GUI version into `godot.exe`, then the renamed exe file will be used for shim or shortcut.

https://github.com/ScoopInstaller/Extras/blob/e0e23474164bf6d97c6990f32029f720a550a241/bucket/godot.json#L16-L26

Scoop seems to automatically determine the app as a GUI app and create shim to run Godot in a new process.

If renamed, we can put exe names in the Scoop manifest `bin` and `shortcut` properties without pre-install commands and use Godot CLI via terminals.

**Comment by Calinou** [comment]
> pre-install commands remove the console version exe

The Scoop manifest can be modified to install this console executable as `godot.console`. The console executable should then automatically detect the main Godot executable this way. (If we want to use `godot-console` instead, Godot could be modified to trim both `.console` and `-console` to find the main executable.)

I wouldn't replace the existing `godot` GUI shim though as its use is sometimes desired.

**Comment by HotariTobu** [comment]
```
"bin": [
    [
        "godot_console.exe",
        "godot"
    ]
],
"shortcuts": [
    [
        "godot.exe",
        "Godot Engine"
    ]
],
```

I think this is the best structure...

Anyway, I would like you to modify the Scoop manifest to leave the console exe and to add a shim for it if possible.
